### DISCUSSION TITLE
Merge Chapters gives a DEMUXER FAILED

### DISCUSSION THREAD

**Discussion by gutomaia** [gopro/labs#479] [General]
I've just  merged two mp4 videos recorded with GoPro MAX.

(Well, first I used the MP4 files downloaded from GoPro Cloud, then I tried using the files splitted from the gopro. Despite the extension, both files are the same, I've MD5sum then.)

Both my videos are using the default chapter size (3.7 GB), I was able to merge the file with ffmpeg with the command:

`ffmpeg -y -f concat -i your_path_to/filelist.txt -c copy -map 0:0 -map 0:1 -map 0:3 -map 0:5 your_output_path/final.mp4`

Then I udtacopy the data from the first mp4 file to the final video.

When I try to open the final result with the GoPro Player, I received the error:

An unexpected error has occured.
Code: DEMUXER_FAILED
Location: MFDemuxer.cpp:28
Message: MFCreateSourceReaderfromByteStream failed 0xc00d36e5

**Comment by dnewman-gpsw** [comment]
This is not a Labs bug, so I moved to discussions.

Are these .MP4s (HERO mode) or .360 files?  If MP4s just use https://github.com/rubegartor/ReelSteady-Joiner/releases as it is much easier than ffmpeg scripting.

**Reply by gutomaia** [reply]
This were recorded in a GoPro MAX, the extension is a MP4 file, although it's a 360 video.

I can provide the files as examples thru DM.

**Comment by GioGio305** [comment]
Si è verificato un errore imprevisto.
Codice: DEMUXER_FAILED
Posizione: MFDemuxer.cpp:28
Messaggio: MFCreateSourceReaderfromByteStream non riuscito 0xc00d36e5
Ho lo stesso probleme, come hai risolto?
